adrs/0000-template.md

```diff
@@ -0,0 +1,16 @@
+# [ADR Title]
+
+**Date:** [YYYY-MM-DD]
+
+**Status:** [ 🚧 Proposed | ✅ Accepted | 🚫 Rejected ]
```

Comment:
Do we need the status field here? Will we ever be committing anything _but_ accepted ADRs? Proposed is a pre-merge concept and rejected never gets merged, which only leaves "accepted" as the sole valid value.

Reply:
Agree. Will remove the status section and the redundancy.